lzy/graph-executor/src/main/resources/db/graph/migrations/V2__Added_failed_task.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE graph_execution_state ADD COLUMN failed_task_id TEXT NULL;
+ALTER TABLE graph_execution_state ADD COLUMN failed_task_name TEXT NULL;
```

Comment:
nit: you may wish to do 2 alters as one: `alter table xxx add column yyy, add column zzz, ....`